### PR TITLE
Sends user identifiers to the events endpoint

### PR DESCRIPTION
**Pull request by wyattjoe40**
* When the `AttentiveConfig.identify()` method is called, the events endpoint is called with the identifiers.

### CODE REVIEW

**Comment by wyattjoe40** [comment]
Not the prettiest async code in the world, but it's abstracted away from the SDK's interface so we can refactor if needed

File: attentive-android-sdk/src/main/java/com/attentive/androidsdk/AttentiveApi.java [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+                return matcher.group(1);
+            }
+        }
+
+        return null;
+    }
+
+    private Metadata buildMetadata(UserIdentifiers userIdentifiers) {
+        Metadata metadata = new Metadata();
+        metadata.setSource("msdk");
```
Comment (oliviakim321) [inline attentive-android-sdk/src/main/java/com/attentive/androidsdk/AttentiveApi.java:142]: nit: do we need this if msdk is already the default source?

File: attentive-android-sdk/src/main/java/com/attentive/androidsdk/AttentiveApi.java [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        try {
+            Metadata metadata = buildMetadata(userIdentifiers);
+            metadataJson = objectMapper.writeValueAsString(metadata);
+        } catch (JsonProcessingException e) {
+            callback.onFailure(String.format("Could not serialize metadata. Message: '%s'", e.getMessage()));
+            return;
+        }
+
+        HttpUrl url = getHttpUrlEventsEndpointBuilder()
+                .addQueryParameter("v", "4.16.10_f0149441bf")
```
Comment (oliviakim321) [inline attentive-android-sdk/src/main/java/com/attentive/androidsdk/AttentiveApi.java:94]: i think we may just be able to use any string for this param - we can test by sending a test event & seeing if it makes it to snowflake

**Review by oliviakim321 — APPROVED** [review]

File: attentive-android-sdk/src/main/java/com/attentive/androidsdk/AttentiveApi.java (hunk 1 above)
Comment (wyattjoe40) [reply attentive-android-sdk/src/main/java/com/attentive/androidsdk/AttentiveApi.java:142]: nope! deleted. thank you

**Comment by wyattjoe40** [comment]
I am closing this PR and making a new one that merges into the `features/identity-improvements` branch.

**Pull request closed without merging**